lib/utils.js

```diff
@@ -71,3 +73,45 @@ export const filterAssets = (scope, assets) => {
             !asset.scope || asset.scope === scope || asset.scope === 'all',
     );
 };
+
+// parse link headers in AssetCss and AssetJs objects
+export const parseLinkHeaders = (headers) => {
+    const links = [];
+
+    if (!headers) return links;
+    headers.split(',').forEach((link) => {
+        const parts = link.split(';');
+        const [href, ...rest] = parts;
+        const value = href.replace(/<|>|"/g, '').trim();
+
+        const asset = { value };
+        for (const key of rest) {
+            const [keyName, keyValue] = key.split('=');
+            asset[keyName.trim()] = keyValue.trim();
+        }
+
+        if (asset['asset-type'] === 'script') {
```

Comment:
detect asset type using fields provided by this PR https://github.com/podium-lib/utils/pull/260